android/sdk/src/main/java/org/jitsi/meet/sdk/ReactActivityLifecycleCallbacks.java

```diff
@@ -126,4 +143,25 @@ public static void onNewIntent(Intent intent) {
             reactInstanceManager.onNewIntent(intent);
         }
     }
+
+    public static void onRequestPermissionsResult(
```

Comment:
All those arguments need to be documented, that includes the 'requestPermissions' method below

Reply:
Or maybe some link to Android documentations about permissions APIs used here. Those arguments kind of come from the Android's Activity#requestPermissions, right ?

Reply:
I'll add some docs. `onRequestPermissionsResult` comes from `Activity` and `requestPermissions` is an extension of Activity's method.